### ISSUE TITLE
Operative zone temperature for discomfort KPI calculation

### ISSUE THREAD

**Issue by JavierArroyoBastida** [ibpsa/project1-boptest#186]
@dhblum this issue is to discuss whether the operative zone temperature should be used instead of (or additionally to) just the air zone temperature that is being used right now for discomfort calculation. Currently, the `KPI_Calculator` integrates the variable sources grouped in AirZoneTemperature of the `kpis.json` file according to the following line:

https://github.com/ibpsa/project1-boptest/blob/95221ca256798efe8acc368f3aaf1c9b31276ee2/kpis/kpi_calculator.py#L109

However, buildings may base their comfort standards and baseline controllers on the operative zone temperature instead. This is particularly critical for buildings with massive thermal mass like thermally activated buildings where operative and air zone temperatures may substantially differ. An example is the multizone commercial hydronic simple test case being developed in https://github.com/JavierArroyoBastida/project1-boptest/tree/issue153_testcaseMultiZoneCommercialHydronic together with @icupeiro, who raised the issue since the baseline controller of this building is based on operative zone temepratures.

My suggestion is:

1. Either we switch to operative zone temperature to compute the discomfort. This would require a few changes in the testcases used for unittesting. 

2. Or we provide the user with the possibility to decide whether to use operative or air zone temperature for discomfort calculation.

**Comment by dhblum** [comment]
@JavierArroyoBastida  @icupeiro I agree operative temperature is important to include for radiant systems, and is the x-axis of the psychrometric chart in ASHRAE Standard 55.  Therefore, it should be included as at least an option.  See actually a previous issue I opened: https://github.com/ibpsa/project1-boptest/issues/113, where I suggested the choice of either air or operative.  My concern about switching to only operative, is that I have not seen it used often as a controlled variable.  Instead, air temperature is much more prevalent.
